File: Semester 4/Advanced Programming/ClientServerApplication/ApplicationNetworking/src/main/java/app/network/dto/DestinationDateDto.java
package app.network.dto;

import java.io.Serializable;
import java.time.LocalDateTime;

public class DestinationDateDto implements Serializable {
    String destination;
    LocalDateTime date;

    @Override
    public String toString() {
        return "DestinationDateDto{" +
                "destination='" + destination + '\'' +
                ", date=" + date +
                '}';
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    public void setDate(LocalDateTime date) {
        this.date = date;
    }

    public String getDestination() {
        return destination;
    }

    public LocalDateTime getDate() {
        return date;
    }

    public DestinationDateDto(String destination, LocalDateTime date) {
        this.destination = destination;
        this.date = date;
    }
}

File: Semester 4/Advanced Programming/RpcProtocolC#/Networking/Dto/DestinationDateDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace network
{
    [Serializable]
    public class DestinationDateDto
    {
        string destination;
        DateTime date;

    public String toString()
    {
        return "DestinationDateDto{" +
                "destination='" + destination + '\'' +
                ", date=" + date +
                '}';
    }

    public void setDestination(String destination)
    {
        this.destination = destination;
    }

    public void setDate(DateTime date)
    {
        this.date = date;
    }

    public String getDestination()
    {
        return destination;
    }

    public DateTime getDate()
    {
        return date;
    }

    public DestinationDateDto(String destination, DateTime date)
    {
        this.destination = destination;
        this.date = date;
    }
}
}
